### PR TITLE
Add CSS loader in WASM

### PR DESCRIPTION
**Pull request by bash**
[Screencast from 2023-02-22 22-29-26.webm](https://user-images.githubusercontent.com/4602612/220763812-d3187025-b91d-4a02-827c-cb009224d445.webm)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -49,7 +49,8 @@ native = [
 
 wasm = [
     "bevy_rapier3d/wasm-bindgen",
-    "core"
+    "core",
+    "wasm-bindgen",
```
Comment (janhohenheim) [inline Cargo.toml:53]: I think this syntax works here:
```suggestion
    "dep:wasm-bindgen",
```

Comment (janhohenheim) [reply Cargo.toml:53]: I realize I could use this syntax in some other optional deps as well, hehe

**Review by janhohenheim — CHANGES_REQUESTED** [review]
You mad lad, haha
I don't like how the UI forms a visual pyramid now. I guess we can't really move the spinner above the browser crashing message, can we?

File: Cargo.toml (hunk 1 above)
Comment (bash) [reply Cargo.toml:53]: Uu what is this syntax?

**Review by janhohenheim — APPROVED** [review]

**Comment by janhohenheim** [comment]
thanks!!

**Pull request merged**